go.mod

```diff
@@ -50,7 +50,7 @@ require (
 	github.com/minio/madmin-go/v2 v2.0.17
 	github.com/minio/minio-go/v7 v7.0.50
 	github.com/minio/mux v1.9.0
-	github.com/minio/pkg v1.6.5
+	github.com/minio/pkg v1.6.6-0.20230330040824-5db111e5f63c
```

Comment:
using `github.com/minio/pkg@main` currently, will change the tag once we make a release of `minio/pkg`